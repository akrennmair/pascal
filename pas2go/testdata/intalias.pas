program test;

type pt = ^intalias;
	intalias = char;

var p : pt;
	i : intalias;

begin
	new(p)
end.
