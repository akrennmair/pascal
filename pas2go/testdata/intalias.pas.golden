package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program test
func main() {
	type (
		intalias byte
		pt       *intalias
	)

	var (
		p pt
		i intalias
	)
	_ = p
	_ = i

	p = new(intalias)
}
